docker-dynamodb/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  db:
+    image: constellation-db
+    deploy:
+      replicas: 1
+      resources:
+        cpus: 0.1
+        memory: 100M
+    volumes:
+      - ./data:/opt/dynamodb/data
```

Comment:
Yeah, the idea is to add a default DB to the repo's data folder.  So when you pull the repo, you get the default database.  After you build the image and run the container any DB updates you do locally get persisted on your computer locally.